jekyll/_cci2/migrating-from-1-2.md

```diff
@@ -45,6 +45,39 @@ CircleCI 2.0 introduces the requirement that you create a configuration file (`.
      ```
 7. Validate your YAML at <http://codebeautify.org/yaml-validator> to check the changes. 
 
+## Steps to Configure Workflows
+
+Optionally configure workflows, using the following instructions:
+
+1. To use the Workflows feature for job orchestration, first split your build job into multiple jobs, each with a unique name.
+
+2. To persist a job artifact, add the `persist_to_workspace:` key at the end of the job, under `steps:`. Then, to get the saved artifact for use by another job, add the `attach_workspace:` key with a nested `at:` key defined as the directory where the artifact was saved.
```

Comment:
~ The `at` property names a directory to which the workspace will be "mounted" (it's not really mounted, contents are copied in). So it doesn't have to be the same directory that was saved via `persist_to_workspace` but it is handy if it is.

E.g. if all jobs use `/tmp/workspace` for the workspace location then everything is nice and consistent.